identity-composer-component.html

```diff
@@ -1,31 +1,137 @@
+<style>
+.text-element {
+  position: relative;
+  left: -45%;
+  font-size: 2.0em;
+  font-weight: bold;
+}
+
+@media (max-width: 1440px) {
+  .text-element {
+    left: -35%;
+    font-size: 1.6em;
+    font-weight: bold;
+  }
+}
+
+@media (max-width: 767px) {
+  .text-element {
+    left: -50%;
+  }
+}
+
+.text-container {
+  position: absolute;
+  margin-top: 1%;
+  margin-left: 7.2%;
+  cursor: pointer;
+}
+
+@media (max-width: 992px) {
+  .text-container {
+    margin-left: 10%;
+  }
+}
+
+@media (max-width: 767px) {
+  .text-container {
+    position: absolute;
+    margin-left: 50%;
+    margin-top: 1%;
+  }
+}
+
+.card-wrapper {
+  position: relative;
+
+}
+
+.info-icon-element {
+
+  position: relative;
+  color: rgba(0, 0, 0, 0.85);
+  right: 1.5%;
+}
+
+.icon-wrapper {
+  right: 0%;
+  top: 2.94%;
+  z-index: 2;
+  background-color: rgba(255, 255, 255, 0.6);
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  border-radius: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.br-identity-profile-container {
+  padding-top: 3.5%;
+}
+
+@media (max-width: 767px) {
+  .br-identity-profile-container {
+    padding-top: 6.5%;
+  }
+
+
+}
+.modal-title {
+  font-size: 26px;
+}
+
+</style>
+
+
 <div ng-if="$ctrl.loading" style="width: 100vw; font-size: x-small;">
   <i class="fa fa-refresh fa-spin text-muted"></i> Building profiles...
 </div>
 <div ng-if="!$ctrl.loading" ng-style="$ctrl.profileContainerStyle">
   <div ng-if="!!$ctrl.profiles.length" class="br-identity-profile-list"
     ng-repeat="profile in $ctrl.profiles">
-    <h4 ng-mouseenter="$hover=true" ng-mouseleave="$hover=false"
-      ng-click="$ctrl.select(profile, $index)">
+<!--     <h4 ng-mouseenter="$hover=true" ng-mouseleave="$hover=false">
       <span class="label label-primary" style="cursor: pointer;">
+        
+      </span>
+    </h4> -->
+ 
+    <div class="text-container">
+      <div class="text-element"
+        style="font-size: calc({{$ctrl.credentialWidth.select}} * 0.07)">
```

Comment:
Could also use angular to do the math vs using `calc()`.
Should `ng-style` be used here?